### PR TITLE
Install Quicksilver Pushback script

### PR DESCRIPTION
**Pull request by ataylorme**
Addresses #4

### CODE REVIEW

**Comment by ataylorme** [comment]
On the Pantheon site the `.gitignore` is not being trimmed so the Quicksilver Pushback script is ignored.

**Comment by greg-1-anderson** [comment]
Looks like your ScriptHandler is missing an autoload entry. I'll add that for you.

File: .gitignore [hunk 1]
```diff
@@ -28,7 +28,6 @@ web/wp
 !web/private/
 web/private/*
 !web/private/scripts/
-web/private/scripts/quicksilver
```
Comment (greg-1-anderson) [inline .gitignore]: We do want to ignore this directory in the canonical repository, though, right?

**Comment by ataylorme** [comment]
> We do want to ignore this directory in the canonical repository, though, right?

Yes, it was still being ignored once sent to Pantheon though and `web/private/scripts/quicksilver` doesn't exist on the multidev so I wanted to test removing the ignore in the canonical repository but it didn't change anything...

**Comment by ataylorme** [comment]
@greg-1-anderson I think the script is there but I am seeing `Expected status 200, got: 403 while running your Quicksilver operation.` with `terminus workflow:watch`. 

`403` seems expected since it's in `web/private`. Not sure if that's stopping the script from processing.

**Comment by ataylorme** [comment]
Looks better now:
```
Enter push-back-to-github. Repository root is /srv/bindings/f9f5640ff5244d648b467dce9e6dd3e2/code.
::::::::::::::::: Build Metadata :::::::::::::::::
array (
  'url' => 'git@github.com:pantheon-systems/example-wordpress-composer.git',
  'ref' => 'issue-4-quicksilver-pushback',
  'sha' => '0ebef10b17257df4725339047377f4217cbc75e9',
  'comment' => 'Move Quicksilver pushback to require-dev and other composer.json cleanup',
  'commit-date' => '2017-07-21 17:33:23 -0400',
  'build-date' => '2017-07-21 21:35:40 +0000',
)

Ignoring commit because it contains build assets.
```

**Comment by ataylorme** [comment]
Tested well. @greg-1-anderson can you review/+1?

**Comment by greg-1-anderson** [comment]
+1 LGTM - good job

**Comment by ataylorme** [comment]
@greg-1-anderson `files/private/github-secrets.json` is created after Quicksilver Pushback runs since `terminus secrets:set` isn't done until after Behat testing. 

This means pushback is failing when attempting after the initial code sync from build assets. I reverted the merge because Quicksilver Pushback throws an undefined index when `files/private/github-secrets.json` doesn't exist.

I'm not convinced we should be doing a pushback on build assets so perhaps a solution here is to skip pushback if `files/private/github-secrets.json` doesn't exist.

**Pull request merged**